acpi/src/madt.rs

```diff
@@ -86,6 +86,10 @@ impl Madt {
             MadtEntry::GicInterruptTranslationService(_) => {
                 unimplemented!();
             }
+
+            MadtEntry::MultiprocessorWakeup(_) => {
```

Comment:
This isn't quite right - we don't want to panic if we encounter one of these. It's probably safest to do nothing in this case, as the MADT should always contain other types of entries (i.e. `MadtEntry::MultiprocessorWakeup(_) => (),`